src/main/kotlin/no/nav/syfo/dialogmotekandidat/api/DialogmotekandidatApi.kt

```diff
@@ -27,10 +29,25 @@ fun Route.registerDialogmotekandidatApi(
                 personIdentToAccess = personIdent,
                 veilederTilgangskontrollClient = veilederTilgangskontrollClient,
             ) {
-                val kandidatDTO: DialogmotekandidatDTO = dialogmotekandidatService.getLatestDialogmotekandidatEndring(
+                val oppfolgingstilfelle = oppfolgingstilfelleService.getSisteOppfolgingstilfelle(
```

Comment:
Her henter man "siste" tilfelle, betyr det at man får det nyeste tilfellet uavhengig av om det er passert eller ikke? Altså man man i teorien kan få en kandidat som ble generert for en måned siden, selv om tilfellet ble avsluttet for 3 uker siden? Det er vel greit uansett, men sikkert lurt å ha et bevist forhold til det 👍

Reply:
Ja, jeg tror det blir slik at tilfellet vil kunne være tilbake i tid, men det kan jo bare være fordi den siste sykmeldingen ikke har kommet inn ennå, så jeg tror ikke vi kan bruke det til å filtrere.

Reply:
Ja, det er sikkert greit. Da må evt. veilederen ta en vurdering på at de vil sette unntak fordi de mener personen ikke er syk lenger 👍